setup.py

```diff
@@ -26,5 +26,12 @@ def configuration(parent_package='',top_path=None):
 if __name__ == "__main__":
     from numpy.distutils.core import setup
 
-    setup(packages=["quicklens","quicklens.cinv","quicklens.qest","quicklens.sims","quicklens.shts","quicklens.math"],
+    packages = ["quicklens","quicklens.cinv","quicklens.qest","quicklens.sims","quicklens.shts","quicklens.math"]
+
+    package_data = dict()
+    data_dir = os.path.join("data","cl","planck_wp_highL")
+    package_data["quicklens"] = [ os.path.join(data_dir,name) for name in os.listdir(os.path.join("quicklens",data_dir)) ]
```

Comment:
I think package_data supports wildcards -- would it work here to simplify with

package_data={'quicklens': ['data/cl/planck_wp_highL/*']}

?